Adds ticket splitting page

s-ben opened the pull request:
Adds a page with an overview of ticket splitting and step-by-step instructions for splitting a ticket.

xaur commented:
I wonder why this PR touches politeia censorship page?

s-ben commented:
@xaur, oops. Looks like I was working off an old branch of master that didn't have the pi censorship page yet? Fixing git mess now...

xaur commented:
Also what is the .pxm file?

matheusd commented on docs/proof-of-stake/ticket-splitting.md:
```diff
@@ -0,0 +1,112 @@
+## Requirements
+
+- **Decrediton wallet:** You will need [Decrediton](../wallets/decrediton/decrediton-setup.md) version 1.2.1 or later to run the Split Ticket Buyer. 
+- **5 DCR:** A minimum of 5 DCR is required to participate in ticket splitting. This must be in your wallet, not on an exchange. 
```
Slightly more than 5 actually, to account for transaction fees.

s-ben replied:
Since the GUI can only do integers, does this mean you have to pledge 6 DCR? Or would the transaction fees be automatically added to the 5 DCR?

matheusd replied:
They are added, so you need an amount _in excess_ of whatever you put as a pledge. For example, if you put in 5 dcr, you'll actually use some 5.003 (amount will actually vary depending on how many inputs you'll actually send once the session starts). So anything above 5.1 should be more than enough.

s-ben replied:
Took a shot at rewording. Is this accurate?

> The minimum amount of DCR you can contribute is 5 DCR. Transaction fees will be automatically calculated by the Split Ticket Buyer and added to your pledged amount when the ticket is purchased. Transaction fees will vary based on the number of participants in the split ticket. A typical transaction fee might be 0.003 DCR. If you have more than 0.1 DCR in your wallet for fees, that should be more than enough.

matheusd replied:
Seems fair enough.

matheusd commented on docs/proof-of-stake/ticket-splitting.md:
```diff
@@ -0,0 +1,112 @@
+For more on ticket splitting, including further explanation of the risks involved, voting, and technical specs of the Split Ticket Buyer software, see the following:
+
+- [DCR Ticket Splitting — All you need to know!](https://medium.com/decred/dcr-ticket-splitting-all-you-need-to-know-b8edc6b65db3): a thorough overview of how ticket splitting works. 
+- [Split Ticket Buyer documentation](https://github.com/matheusd/dcr-split-ticket-matcher/tree/master/docs): Technical documentation for the Split Ticket Matcher software used to fascilitate ticket splitting. 
```
typo on `fascilitate`

s-ben replied:
Fixed

matheusd commented on docs/proof-of-stake/ticket-splitting.md:
```diff
@@ -0,0 +1,112 @@
+
+1. Download the Split Ticket Matcher software for your OS.
+
+    - Split Ticket Matcher [downloads](https://github.com/matheusd/dcr-split-ticket-matcher/releases/tag/v0.7.0).
```
Should probably link to the "releases" page instead of a specific tag.

s-ben replied:
Done

matheusd commented on docs/proof-of-stake/ticket-splitting.md:
```diff
@@ -0,0 +1,112 @@
+
+    ![Split Ticket Buyer config menu](../img/ticket_splitting_joined_session.png)
+
+    You are now in the ticket splitting pool! Until the ticket price is reached, no DCR is spent. Participants are free to change the amount pledged, change wallets, or leave the pool altogether. 
```
Actually, you can't change the amount before stopping participation, then restarting (changing the slider after you clicked on "Participate" doesn't really do anything).

s-ben replied:
Reworded to:

> You are now in the ticket splitting pool! Until the ticket price is reached, no DCR is spent. Participants are free to the leave pool at any time. They can also re-enter the pool using a different wallet or pledging a different amount of DCR.

matheusd replied:
Ok on the reword, but seems you haven't pushed it yet.

s-ben replied:
Pushed. Have been wondering about the etiquette around that. Sometimes I push suggested changes. Sometimes I don't, with the reasoning that I don't want to "spam" the git history with too many commits. But actually, from a security perspective you have to push I suppose.

matheusd replied:
This is really going to depend on each project, but in general you shouldn't worry about "spam" in git history.

Before actually merging the changes from a PR into master usually all commits on it will be merged into a single final commit, squashing all changes and removing the "noise" commits. Pushing individual commits into the PR itself is preferable in this case, since reviewers can see the individual changes.

s-ben replied:
Right. I do that on code commits to other repos, but here on docs, it's a bit looser. Some are git noobs, and I guess the norm anyway is that a clean commit tree isn't as important with docs.

s-ben replied:
Thanks for your quick help reviewing!

s-ben commented:
@matheusd, have addressed all of your comments, but you'll probably want to review a couple of my rewordings (pasted in comments for convenience) for accuracy.

xaur commented:
@s-ben I could view all png files but not the `split_ticket_buyer_amount.pxm` and it's not linked anywhere in the doc. What is pxm?

s-ben commented:
@xaur, good catch. That `.pxm` file was a source file from my poor man's Photoshop Pixelmator. Removed. 

@matheusd have pushed suggested changes. Should be good to go!
